pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -398,6 +428,23 @@ func (manager *Manager) reconcileCluster(cluster *clusterv1alpha2.PediaCluster)
 	return controller.NoRequeueResult
 }
 
+func (manager *Manager) reconcileShardingUnmanaged(cluster *clusterv1alpha2.PediaCluster) controller.Result {
+	if *cluster.Status.ShardingName == manager.shardingName {
+		if err := manager.removeCluster(cluster.Name); err != nil {
```

Comment:
removeCluster should not be called, it will clean up the data in the storage tier, we just need to stop synchro, we don't need to delete the data.